README.md

```diff
@@ -32,7 +32,7 @@ Quick links:
 | [DDI-PDF](https://github.com/gdcc/exporter-ddipdf)          | Export DDI metadata as a printable PDF file                                                                                                  | PDF with [DDI metadata](https://ddialliance.org/) | pdf            | TBD      |
 | [Croissant](https://github.com/gdcc/exporter-croissant)     | Export metadata as linked data following [Croissant ontology](https://docs.mlcommons.org/croissant/docs/croissant-spec.html)                 | JSON-LD                                           | croissant      | [JAR file](https://repo1.maven.org/maven2/io/gdcc/export/croissant/) |
 | [RO-Crate](https://github.com/gdcc/exporter-ro-crate)       | Export metadata as linked data following [RO-Crate ontology](https://www.researchobject.org/ro-crate/specification/1.1/appendix/jsonld.html) | JSON-LD                                           | rocrate_json   | TBD      |
-| [Transformer](https://github.com/gdcc/exporter-transformer) | This exporter allows you to have up to 100 exporters using a single pre-built JAR file. Included examples: Hello World!, Debug, Croissant, RO-Crate, DDI-PDF, and more! You can add new exporters by adding directories into the exporters directory and placing (and editing) the `config.json` and the `transformer` files in it (see the [README](https://github.com/gdcc/exporter-transformer?tab=readme-ov-file)). | JSON(-LD), XML, HTML, PDF, etc. | (varies)       | [JAR file](https://repo1.maven.org/maven2/io/gdcc/export/exporter-transformer/1.0.9/exporter-transformer-1.0.9-jar-with-dependencies.jar) |
+| [Transformer](https://github.com/gdcc/exporter-transformer) | This exporter allows you to have up to 100 exporters using a single pre-built JAR file. Included examples: Hello World!, Debug, Croissant, RO-Crate, DDI-PDF, and more! You can add new exporters by adding directories into the exporters directory and placing (and editing) the `config.json` and the `transformer` files in it (see the [README](https://github.com/gdcc/exporter-transformer?tab=readme-ov-file)). | JSON(-LD), XML, HTML, PDF, etc. | (varies)       | [JAR file](https://repo1.maven.org/maven2/io/gdcc/export/exporter-transformer/1.0.10/exporter-transformer-1.0.10-jar-with-dependencies.jar) |
```

Comment:
```suggestion
| [Transformer](https://github.com/gdcc/exporter-transformer) | This exporter allows you to have up to 100 exporters using a single pre-built JAR file. Included examples: Hello World!, Debug, Croissant, RO-Crate, DDI-PDF, and more! You can add new exporters by adding directories into the exporters directory and placing (and editing) the `config.json` and the `transformer` files in it (see the [README](https://github.com/gdcc/exporter-transformer?tab=readme-ov-file)). | JSON(-LD), XML, HTML, PDF, etc. | (varies)       | [JAR file](https://repo1.maven.org/maven2/io/gdcc/export/exporter-transformer/) |
```

What if we simply link to the directory and let people find the latest JAR file? It would make maintaining this README easier.